includes/Modules/Analytics_4.php

```diff
@@ -132,14 +134,21 @@ public function get_debug_fields() {
 		$settings = $this->get_settings()->get();
 
 		return array(
-			// TODO: This can be uncommented when Analytics and Analytics 4 modules are officially separated.
-			/* // phpcs:ignore Squiz.PHP.CommentedOutCode.Found
+			// phpcs:disable
```

Comment:
I couldn't not get PHPCS to be happy here due to various conflicting rules so needed to disable it for this block.